install.sh

```diff
@@ -65,54 +57,51 @@ else
   echo_info "Unable to detect system information, installation will continue."
 fi
 
+# Check if the script is being run as root
 if [ "${EUID}" == "0" ]; then
   echo_error "Chromebrew should not be installed or run as root."
   exit 1;
 fi
 
 echo_success "Welcome to Chromebrew!"
 
-# prompt user to enter the sudo password if it set
-# if the PASSWD_FILE specified by chromeos-setdevpasswd exist, that means a sudo password is set
+# Prompt user to enter the sudo password if it is set
+# If the PASSWD_FILE specified by chromeos-setdevpasswd exist, that means a sudo password is set
 if [[ "$(< /usr/sbin/chromeos-setdevpasswd)" =~ PASSWD_FILE=\'([^\']+) ]] && [ -f "${BASH_REMATCH[1]}" ]; then
   echo_intra "Please enter the developer mode password"
-  # reset sudo timeout
+  # Reset sudo timeout
   sudo -k
   sudo /bin/true
 fi
 
-# force curl to use system libraries
+# Force curl to use system libraries
 function curl () {
-  # retry if download failed
-  # the --retry/--retry-all-errors parameter in curl will not work with the 'curl: (7) Couldn't connect to server'
-  # error, a for loop is used here
+  # Retry if download failed
+  # The --retry/--retry-all-errors parameter in curl will not work with the 'curl: (7) Couldn't connect to server' error,
+  # So a for loop is used here
   for (( i = 0; i < 4; i++ )); do
     env LD_LIBRARY_PATH='' ${CURL} --ssl -C - "${@}" && \
       return 0 || \
       echo_info "Retrying, $((3-$i)) retries left."
   done
-  # the download failed if we're still here
+  # The download failed if we're still here
   echo_error "Download failed :/ Please check your network settings."
   return 1
 }
 
-case "${ARCH}" in
-"i686"|"x86_64"|"armv7l"|"aarch64")
-  LIB_SUFFIX=
-  [ "${ARCH}" == "x86_64" ] && LIB_SUFFIX='64'
-  ;;
-*)
-  echo_error "Your device is not supported by Chromebrew yet :/"
-  exit 1;;
-esac
-
 echo_info "\n\nDoing initial setup for install in ${CREW_PREFIX}."
 echo_info "This may take a while if there are preexisting files in ${CREW_PREFIX}...\n"
 
-# This will allow things to work without sudo
-crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
+crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var
+              etc/crew/meta tmp/crew"
```

Comment:
How about `etc/crew` instead of `etc/crew/meta`?

Reply:
`etc/crew/meta` is required for some lines in `extract_install`.

Reply:
Right but `etc/crew` will include all subfolders when removed which would be more thorough.

Reply:
Removed? Not sure what you mean. This just creates and takes ownership of missing folders in CREW_PREFIX.

Reply:
You respond too quick.  I made a correction.

Reply:
Ah. `etc` includes the `etc/crew` subfolder (and thus the subfolders within it), does it not?

Reply:
Right but we should only change crew folder permissions.

Reply:
Actually, disregard everything I said and leave this unchanged.  We are making directories if they don't exist.